ros/processors/archive_processor.py

```diff
@@ -0,0 +1,78 @@
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+import requests
+
+from insights import extract, run, rule, make_metadata
+from insights.parsers import uname
+from io import open
+import sys
+
+
+@contextmanager
+def unpacked_remote_archive(url):
+    """ Work with remote archive ( download + unpack )"""
+    try:
+        with NamedTemporaryFile(delete=True) as file:
```

Comment:
I guess by default `NamedTemporaryFile` has `delete=True`. Do we need to mention it explicitly?